demo/src/App.jsx

```diff
@@ -207,6 +204,23 @@ class App extends Component {
             <Section size="md">
               <Table stackAt="max-md" data={tableData} />
             </Section>
+            <Pagination.Wrapper activeLabel="Pagination Navigation">
+              <Pagination.Item>
+                <Pagination.Link className="c-pagination__link--previous" to="/" disabled>Prev</Pagination.Link>
+              </Pagination.Item>
+              <Pagination.Item>
+                <Pagination.Link className="c-pagination__link" to="/">1</Pagination.Link>
```

Comment:
This className shouldn't have to be specified by the consumer.

Reply:
done